app/models/repo.rb

```diff
@@ -3,6 +3,7 @@ class Repo
   def initialize(gh_response)
     @url = gh_response["html_url"]
     @name = gh_response["name"]
+    @gh_response = gh_response
```

Comment:
we can get rid of this guy too

Reply:
yup. leftover from when we had less of an idea of how to work with json.